healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java

```diff
@@ -74,7 +74,7 @@ public static void fhirResourceGetHistory(String resourceName, String versionId)
       responseEntity.writeTo(System.err);
       throw new RuntimeException();
     }
-    System.out.println("FHIR resource history list retrieved: ");
+    System.out.println("FHIR resource retrieved from version: ");
```

Comment:
Does the test check for this text in the output?

Reply:
Just fixed the test, thanks.